profile/README.md

```diff
@@ -4,21 +4,10 @@
 ![orkes logo](https://user-images.githubusercontent.com/1514288/155636248-0b60f994-4b32-4b53-b061-6904c2e413ac.png#gh-dark-mode-only)
 
 
-## Orkes Platform is a cloud hosted, fully managed version of Netflix Conductor offering full compatibility with the open source version.
+## Orkes Platform is a cloud hosted, fully managed version of Conductor OSS - a platform to orchestrate microservices and events [originally created and maintained by Netflix](https://github.com/Netflix/conductor). 
```

Comment:
probably can remove `maintained` at this point to avoid being ambiguous (they are not maintaining it anymore at all)

Reply:
we should link this to https://github.com/conductor-oss/conductor.  Lets remove Netflix references.